README.md

```diff
@@ -68,8 +72,8 @@ simple
 ```
 
 **In this project directory, do the following:**
-1) >npm install --save-dev eslint-plugin-i18n-json 
-2) Create a `.eslintrc.js` file in the root dir of your project.
+1) >npm install --save-dev eslint-plugin-i18n-json
```

Comment:
is the block quote intentional?  do you want fixed width instead?

Reply:
yes, it's intentional, I wanted to make terminal commands have a distinct look